gst/isomp4/qtdemux.c

```diff
@@ -3050,6 +2965,11 @@ qtdemux_parse_moof (GstQTDemux * qtdemux, const guint8 * buffer, guint length,
   g_node_destroy (moof_node);
   return TRUE;
 
+cenc_parse_error:
```

Comment:
label cenc_parse_error is unused (if we don't enable fluc-sdk), on new versions of gcc compilation will fail